packages/utils/src/lib/reports/__snapshots__/log-stdout-summary.integration.test.ts.snap

```diff
@@ -96,26 +94,25 @@ exports[`generateStdoutSummary > should not contain category section when catego
 ESLint audits
 
 ● Disallow missing props validation in a React component definition   6 warnings
-● Disallow variable declarations from shadowing variables declared    3 warnings
-  in the outer scope
-● Require or disallow method and property shorthand syntax for        3 warnings
-  object literals
+● Disallow variable declarations from shadowing variables declared in 3 warnings
+the outer scope
+● Require or disallow method and property shorthand syntax for object 3 warnings
+literals
 ● verifies the list of dependencies for Hooks like useEffect and      2 warnings
-  similar
+similar
 ● Disallow missing \`key\` props in iterators/collection literals       1 warning
 ● Disallow unused variables                                           1 warning
 ● Enforce a maximum number of lines of code in a function             1 warning
-● Require \`const\` declarations for variables that are never           1 warning
-  reassigned after declared
+● Require \`const\` declarations for variables that are never reassigned1 warning
+after declared
```

Comment:
:lady_beetle: Padding between columns is lost :cry:

Reply:
THX for spotting should be fixed now